includes/elements/section.php

```diff
@@ -1530,7 +1530,7 @@ private function print_shape_divider( $side ) {
 		$negative = ! empty( $settings[ $base_setting_key . '_negative' ] );
 		?>
```

Comment:
Looks good but we need an extra check if the file actually exists something like 
```suggestion
		$shape_path = Shapes::get_shape_path( $settings[ $base_setting_key ], $negative );
		if ( ! file_exists( $shape_path ) ) {
			return;
		}
		?>
```